yeat/tests/test_cli.py

```diff
@@ -41,7 +42,7 @@ def test_basic_dry_run(tmp_path):
 
 
 def test_no_args():
-    with pytest.raises(SystemExit, match=r"2"):
+    with pytest.raises(Exception, match=r"yeat is not an integer"):
```

Comment:
This test is unclear. Where is `"yeat"` being passed to an argument where an integer is expected? How is this behavior relevant to `yeat` being run with no arguments?

Reply:
Reverted back to the original test. The reason for this exception test is because when we call the cli with no arguments, in the command line, `yeat` is considered a CLI argument. I do understand why this is the case.

For example, I were to type `yeat` into the terminal, I would get this error:
>usage: yeat [-h] [-v] [-o DIR] [-t T] [--sample S] [-n] [-d D] [-c C] [--init] config reads reads
yeat: error: the following arguments are required: config, reads

When I run the cli with no input, `cli.main(None)`, I get the following error:
usage: yeat [-h] [-v] [-o DIR] [-t T] [--sample S] [-n] [-d D] [-c C] [--init] config reads reads
yeat: error: the following arguments are required: reads

Notice that `config` is missing the error message above.

The reason why the outdated test had a regex string `yeat is not an integer` is because `yeat` would be considered a string in the check positive function.

I am unclear how this is the case, but this is the way the argparse parses a empty string command.

Reply:
UPDATE: Test was removed.